charts/plex/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.19.1.2645-ccb6eb67e
 description: Plex Media Server
 name: plex
-version: 1.5.2
+version: 1.5.3
```

Comment:
As we're adding a new 'feature' here, can you please bump the chart version to 1.6.0?

Reply:
See if that space fixes that EOF. I have had weird errors in other languages due to the ATOM editor doing something invisible. Normally doing something stupid like adding a space fixes it